gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go

```diff
@@ -245,7 +247,9 @@ func (p *glusterBlockProvisioner) createSecretRef(nameSpace string, secretName s
 		Type: chapType,
 	}
 
+	p.volConfig.BlockSecret = secretName
 	secretRef := &v1.LocalObjectReference{}
+	p.volConfig.BlockSecretNs = nameSpace
 	if secret != nil {
 		_, err = p.client.Core().Secrets(nameSpace).Create(secret)
```

Comment:
What if the secret already exists? Isn't there a very high chance of that if Provisioning multiple PVs? I was under the impression they all use the same secret (storage class immutable)